docs/design/fabric-v2+/fpc-registration.puml

```diff
@@ -73,7 +84,16 @@ group enclave creation
     See fpc-components.uml diagram for more details
   end note
 
-  Enclave1 -> Enclave1 : Attested_Data <- <CC_Parameters, Public_Keys>
+  Enclave1 -> Enclave1 : Attested_Data <- <CC_Parameters, Public_Keys, onCreation=true>
+  note right Enclave1
+     In the future an admin might want to request re-attestations: For EPID this would be
+     the only way to check for revocations (In DCAP, checking CRLs would enable this as well.)
```

Comment:
```suggestion
     the only way to check for revocations (In DCAP, checking CRLs would enable this as well).
```

Reply:
@mbrandenburger the full stop inside the parenthesis was actually correct (as it is a full sentence). That was missing though was a full stop (or probably better semi-colon) behind the previous sentence on EPID before :-)     But thanks a lot for the careful review and, in particular, catching the `s/, /./` in  `Credential, Certificate`  without the semantics would have been confusing.  (I thought i've done that but i guess it got somewhat lost, maybe also in my aborted attempt to change the "." to "::" for field dereferences as strictly speaking UML would require ...